### ISSUE TITLE
the issues of language

### ISSUE THREAD

**Issue by touchingwang** [doublespeakgames/adarkroom#715]
In the latest 1.4 version,A Dusty Path,when exploring to 'X',Events appear in English only.

**Comment by deltamoins** [comment]
Indeed, I'm playing the French version and just have seen that. I thouhgt it was a lack of translation and was ready to help but if it's not that...

**Comment by Continuities** [comment]
The expansion content is indeed untranslated!

**Comment by deltamoins** [comment]
OK, do you need help to do it ? I'm new to that but I perhaps try to help you.

**Comment by Continuities** [comment]
I certainly don't have the fluency in French to translate, so if you want
to take it on I'd welcome the help <3

**Comment by deltamoins** [comment]
OK, I'll try my best. As I said, I've never done that before although I've always wanted to participate as a Linux and GNU software user.
Here's what I propose to you: I first finish the game and then I work on the missing translation in French. I prefer to do so to avoid getting spoiled about the content of the untranslated part.
Does it seem OK for you ?

**Comment by Continuities** [comment]
Whatever you like 😊
